examples/guestbook/v1beta3/redis-slave-controller.json

```diff
@@ -0,0 +1,39 @@
+{
+    "kind": "ReplicationController",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "redis-slave",
+        "labels": {
+            "name": "redis-slave"
```

Comment:
kindly request to keep redis-master as is to keep back ward compatibility with initial brendanburns version